_plugins/import_tag.rb

```diff
@@ -1,30 +1,11 @@
-# This code is roughly based on https://github.com/koraktor/jekyll/tree/import-tag
-#
 # frozen_string_literal: true
 
+# TODO: this still exists only to avoid a crash with jekyll-feed, and we should find a better solution. This can mean ensuring jekyll-feed skips nodes, or choosind a different syntax for imports, or...
```

Comment:
I found a better option in setting out hook with a higher priority than `jekyll-feed`'s. I have to test it yet, but it should work.

Still, using the syntax of a liquid tag might be misleading.
Maybe something similar to a "preprocessor directive" like `#include 000F` would be better.

Reply:
Gotcha! My only thought is that `#include` might be a bit confusing because of Markdown. What about `@include 000F`? I think this doesn't conflict with anything.